File: Java/String/program8.java
//Accept the string from user and display how many times the characters 'l' contains in the string

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public int CountL()
    {
        char Arr[] = str.toCharArray();

        int i = 0;
        int iCnt = 0;

        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] == 'l')
            {
                iCnt++;
            }
        }
        return iCnt;
    }
}

class program8
{
    public static void main(String arg[])
    {
        int iRet = 0;

        Stringx obj = new Stringx();

        obj.Accept();

        iRet = obj.CountL();
        System.out.println("String contains l : "+iRet + " times");
    }
}

File: C/String/program8.c
//Accept the string from user and display its length

#include<stdio.h>

int StrlenX(char str[])
{
    int iCnt = 0;

    while(*str != '\0')
    {
        iCnt++;
        str++;
    }
    return iCnt;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    iRet = StrlenX(Arr);
    printf("Length of string is : %d",iRet);

    return 0;
}
